packages/node-resolve/src/fs.js

```diff
@@ -2,8 +2,15 @@ import fs from 'fs';
 
 import { promisify } from 'util';
 
-export const exists = promisify(fs.exists);
 export const readFile = promisify(fs.readFile);
 export const realpath = promisify(fs.realpath);
 export { realpathSync } from 'fs';
 export const stat = promisify(fs.stat);
+export async function exists(filePath) {
```

Comment:
Since `fs.exists` is just a wrapper around `fs.access` you should probably use that to make sure the behaviour is the same and to not unnecessarily create the stat object just to throw it away

https://github.com/nodejs/node/blob/5ff2e582cd867feba71bb13d9d8bed9953721145/lib/fs.js#L240-L252